packages/world-modules/src/modules/utils/SystemSwitch.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.21;
```

Comment:
talked IRL but can we move this to `src/utils` instead of `stc/modules/utils` since it's not a module?